source/adios2/toolkit/transport/wan/WANZmq.cpp

```diff
@@ -51,7 +56,7 @@ void WANZmq::Open(const std::string &name, const OpenMode openMode)
 
         m_Socket = zmq_socket(m_Context, ZMQ_REQ);
         const std::string fullIP("tcp://" + m_IPAddress + ":" + m_Port);
-        zmq_connect(m_Socket, fullIP.c_str());
+        int err = zmq_connect(m_Socket, fullIP.c_str());
```

Comment:
Does err need to be checked?  What happens if zmq_connect fails?

Reply:
@JasonRuonanWang @chuckatkins makes a good point. Let's address this with a runtime exception check that is always on. On a side note, this is what I mean by avoiding the underscore, just by looking at zmq_connect I know it's coming from a library.